e2e/cypress/integration/team/teams_spec.js

```diff
@@ -137,9 +137,11 @@ describe('Teams Suite', () => {
 
         leaveAllTeams();
 
-        cy.get('a.signup-team-login').should('contain', 'Create a new team');
```

Comment:
Failing log: Timed out retrying: Expected to find element: 'a.signup-team-login', but never found it. ([link](https://build-push.internal.mattermost.com/job/mattermost-ui-testing/job/mattermost-cypress/97/testReport/(root)/TS14633%20Leave%20all%20teams/Teams_Suite_TS14633_Leave_all_teams/))

Such element is dependent on some permission level.  The [test case](https://docs.google.com/spreadsheets/d/14qGnjguYJft7vBTzc7MK0Uai50s1NY9xfD9KFC1GDk0/edit#gid=822208765&range=C14) actually require to check selection page and so checking URL redirect is sufficient enough for it.